8.x/getting-started/documentation.md

```diff
@@ -2,4 +2,4 @@
 
 [[toc]]
 
-under construction.
\ No newline at end of file
+This open source project is being documented by developers well trained under the supervision of an expert team leader.
```

Comment:
Hi, please change the line with the paragraph below:
Larammerce is getting documented, as well as developed at high speed. So if you want to be on track, follow the guides.

Reply:
OK . I did the job .